modules/client/src/lib/constants.ts

```diff
@@ -36,6 +36,8 @@ export const BOOTY = {
   type: CurrencyType.BEI,
 }
 
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
the utils file in the client exports an emptyAddress and emptyRootHash (uses it for some generation fns). should probably keep all the constants in here though, since it looks like the hub is importing the constants file from the lib of the client as well